datacontract/export/sql_type_converter.py

```diff
@@ -226,9 +226,10 @@ def convert_to_duckdb(field: Field) -> None | str:
         "bigint": "BIGINT",
         "date": "DATE",
         "time": "TIME",
-        "timestamp": "TIMESTAMP WITH TIME ZONE",
+        "datetime": "TIMESTAMP",
+        "timestamp": "TIMESTAMP",
```

Comment:
timestamp is a synonym with timestamp_tz and should have the same type

Reply:
plus `datetime` is not a valid Data Contract Data Typehttps://datacontract.com/#data-types